docs/tutorials/contributing/strategy/writing_test_for_the_new_strategy.rst

```diff
@@ -6,10 +6,39 @@ where :code:`<library>.py` is the name of the file you have created or similarly
 add tests to the test file that is already present in the
 :code:`axelrod/tests/unit/` directory.
 
+Typically we want to test the following:
+
+* That the strategy behaves as intended on the first move and subsequent
+  moves, triggering any expected actions
+* That the strategy initializes correctly
+* That the strategy resets and clones correctly
+
+If the strategy does not use any internal variables then there are generic tests
+that are automatically invoked to verify proper initialization, resetting, and
+cloning.
+
+There are several convenience functions to help write tests efficiently for
+how a strategy plays.
+
+* `first_play_test(action, seed=None)` tests the strategy's first action, taking
+  an optional random seed in case the strategy is stochastic. If so, please
+  include cases where both outcomes are observed, e.g.::
+
+    def test_strategy(self):
+        self.first_play_test(C, seed=11)
+        self.first_play_test(D, seed=23)
+
+* `second_play_test(rCC, rCD, rDC, rDD, seed=None)` tests the strategies actions
```

Comment:
Yes the extra example (that was not the point I was making). You introduce these functions twice, from a newcomer/beginner's point of view that can be confusing. 

I'll write a modification and send you a PR and you can see what I'm trying to say.